allauth/account/tests/test_reset_password.py

```diff
@@ -300,3 +300,13 @@ def _create_user_and_login(self, usable_password=True):
     def _password_set_or_change_redirect(self, urlname, usable_password):
         self._create_user_and_login(usable_password)
         return self.client.get(reverse(urlname))
+
+def test_password_change_ACCOUNT_LOGOUT_ON_PASSWORD_CHANGE(settings, auth_client, user_password):
+    settings.ACCOUNT_LOGOUT_ON_PASSWORD_CHANGE = True
+    data = {
+        "oldpassword": user_password, 
+        "password1": "newpass123", 
+        "password2": "newpass123"
+    }
+    resp = auth_client.post(reverse("account_change_password"), data)
+    assert resp['location'] == reverse("account_login")
```

Comment:
The build is failing on this point. Shouldn't you assert on `== settings.ACCOUNT_LOGOUT_REDIRECT_URL` ?

Reply:
Exactly yes, I mixed my project's settings for the defaults